src/md/compiler/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(MDCOMPILER_SOURCES
   import.cpp
   importhelper.cpp
   mdutil.cpp
+  newmerger.cpp
```

Comment:
The new merged is pretty straightforward to delete: https://github.com/jkotas/coreclr/commit/cf82a1007283bd1923b095ea225f83931dd9835e